sdk/state/open.go

```diff
@@ -67,8 +88,7 @@ func (c *Channel) OpenTxs(d OpenAgreementDetails) (txClose, txDecl, formation *t
 		InitiatorEscrow: c.initiatorEscrowAccount().Address,
 		ResponderEscrow: c.responderEscrowAccount().Address,
 		StartSequence:   c.startingSequence,
-		Asset:           d.Asset,
-		AssetLimit:      d.AssetLimit,
+		Assets:          d.Assets,
```

Comment:
I think this field should be TrustLines too.

Reply:
https://github.com/stellar/experimental-payment-channels/pull/139/commits/cf545163cc011dcbf4911d713363b4a65ad8a64e